jpo-ode-common/src/main/java/us/dot/its/jpo/ode/inet/InetPacketSender.java

```diff
@@ -20,128 +20,141 @@
 import java.net.DatagramSocket;
 import java.net.SocketException;
 
-import org.apache.log4j.Logger;
+import us.dot.its.jpo.ode.eventlog.EventLogger;
 
 /**
- * Sender/Forwarder helper class for use by Forwarder, Transport, and Data Sink that need to send packets around
+ * Sender/Forwarder helper class for use by Forwarder, Transport, and Data Sink
+ * that need to send packets around
  */
 public class InetPacketSender {
-	
-	private static final String INVALID_PARAMETERS_MSG = "Invalid Parameters. Parameters destination point and payload can not be null";
 
-  private static final Logger log = Logger.getLogger(InetPacketSender.class);
```

Comment:
That class (and others with the same syntax, including the EventLogger used here) is using slf4j rather than log4j, the ["Simple Logging Facade for Java"](http://www.slf4j.org/manual.html), which acts as a wrapper for other concrete logging instances. In our case, that concrete instantiation is logback. It looks like usage between the custom EventLogger and the default slf4j usage of `LoggerFactory.getLogger(this.getClass())` is about equal. I'm not sure why this wrapper exists but it does seem to be used quite a bit.

Reply:
Thanks!